saq/queue.py

```diff
@@ -228,12 +249,18 @@ async def listen(self, job, callback, timeout=10):
         """
         Listen to updates on job.
 
-        job: job instance
+        job: job instance or an iterable of job instances
```

Comment:
can you change it to be jobs?

Reply:
I was worried this would be a breaking change (if anyone is calling this with the kwarg `job`).

Happy to change it if you're not worried about that, though

Reply:
we'll just bump the version, it's fine